installer.sh.in

```diff
@@ -837,7 +849,16 @@ configure_wifi() {
     fi
 
     rm -f ${_wpasupconf%.conf}-${dev}.conf
-    cp -f ${_wpasupconf} ${_wpasupconf%.conf}-${dev}.conf
+
+    # wpa_supplicant defaults
+    echo "ctrl_interface=/run/wpa_supplicant" >> ${_wpasupconf%.conf}-${dev}.conf
+    echo "ctrl_interface_group=wheel"         >> ${_wpasupconf%.conf}-${dev}.conf
+    echo "eapol_version=1"                    >> ${_wpasupconf%.conf}-${dev}.conf
+    echo "ap_scan=1"                          >> ${_wpasupconf%.conf}-${dev}.conf
+    echo "fast_reauth=1"                      >> ${_wpasupconf%.conf}-${dev}.conf
+    echo "update_config=1"                    >> ${_wpasupconf%.conf}-${dev}.conf
```

Comment:
Why hardcoded here?

Reply:
Because I want ${_wpasupconf} to be a symlink to ${_wpasupconf%.conf}-${dev}.conf. On the root filesystem, the default wpa_supplicant configuration is stored ${_wpasupconf}. When the installer runs, ${_wpasupconf} is copied to ${_wpasupconf%.conf}-${dev}.conf, and then options are appended to the latter file. However, if ${_wpasupconf} is used as symlink to the wpa_supplicant configuration file for the configured device, then the default wpa_supplicant options have to be stored somewhere else. I just hardcoded them in the installer, but one could easily just create /etc/wpa_supplicant/wpa_supplicant.defaults, and put the default options there.

Reply:
I still don't understand why these options are hardcoded here.